modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/SqlQueryProcessor.java

```diff
@@ -512,15 +497,11 @@ private AsyncSqlCursor<List<Object>> executePlan(
 
                     @Override
                     public CompletableFuture<BatchedResult<List<Object>>> requestNextAsync(int rows) {
-                        session.touch();
```

Comment:
can we remove `session` from the parameters of the `executePlan` method?

Reply:
removed, thanks!